vendors/renesas/boards/rx65n-rsk/CMakeLists.txt

```diff
@@ -0,0 +1,273 @@
+# Set variables.
+set(renesas_dir "${AFR_VENDORS_DIR}/renesas")
+set(rx65nrsk_dir "${AFR_VENDORS_DIR}/renesas/boards/rx65n-rsk")
+
+set(rx65nrsk_ports_dir "${rx65nrsk_dir}/ports")
+
+if(AFR_IS_TESTING)
+    set(rx65nrsk_aws_dir "${rx65nrsk_dir}/aws_tests")
+    set(exe_target aws_tests)
+else()
+    set(rx65nrsk_aws_dir "${rx65nrsk_dir}/aws_demos")
+    set(exe_target aws_demos)
+endif()
+
+# -------------------------------------------------------------------------------------------------
+# Amazon FreeRTOS Console metadata
+# -------------------------------------------------------------------------------------------------
+
+afr_set_board_metadata(ID "Renesas-RX65N-RSK")
+afr_set_board_metadata(DISPLAY_NAME "Renesas Starter Kit+ for RX65N-2MB")
+afr_set_board_metadata(DESCRIPTION "The Renesas Starter Kit+ for RX65N-2MB connects to AWS IoT Cloud using Ethernet")
+afr_set_board_metadata(VENDOR_NAME "Renesas")
+afr_set_board_metadata(FAMILY_NAME "RX65N")
+afr_set_board_metadata(CODE_SIGNER "null")
+afr_set_board_metadata(DATA_RAM_MEMORY "640KB")
+afr_set_board_metadata(PROGRAM_MEMORY "2MB")
+afr_set_board_metadata(SUPPORTED_IDE "E2Studio")
+afr_set_board_metadata(IDE_E2Studio_NAME "e2 studio")
+afr_set_board_metadata(IDE_E2Studio_COMPILER "CCRX")
+afr_set_board_metadata(IDE_E2Studio_RECOMMENDED "true")
+
+# -------------------------------------------------------------------------------------------------
+# Compiler settings
+#
+# Note1: Cmake doesn't support cc-rx compiler. The settings are commented out.
+# Note2: The settings below are NOT test yet. Some flags might be wrong and some might be missing.
+# -------------------------------------------------------------------------------------------------
+afr_mcu_port(compiler)
+
+#set(compiler_flags
```

Comment:
You should remove the parts that are commented out, just like with code.

Reply:
Actually we don't need to comment those out, just leave them here, the notes above is suffice imo. Since we're not building it with CMake anyway.